zqd/packet/packet.go

```diff
@@ -74,64 +74,83 @@ func IngestFile(ctx context.Context, s *space.Space, pcap, zeekExec string) (*In
 		done:      make(chan struct{}),
 		zeekExec:  zeekExec,
 	}
+	if err = p.indexPcap(ctx); err != nil {
+		os.Remove(p.space.DataPath(IndexFile))
+		return nil, err
+	}
+	if err = p.space.SetPacketPath(p.pcapPath); err != nil {
+		os.Remove(p.space.DataPath(IndexFile))
+		return nil, err
+	}
 	go func() {
 		p.err = p.run(ctx)
 		close(p.done)
 	}()
-	return p, p.awaitZeekLogs()
```

Comment:
My ideal would be that the inner `writeData` loops waits to kick off until after log files... that said I think for now it's ok to get rid of this functionality as well `fsnotify`. If you are doing this however, I think `writeData` needs to be adjusted so it doesn't return an error in the event that no long files exist.

Reply:
My intention was that once the client got a `200` response code, it would mean there was data available for the client to query. The `status` api will need to change to somehow notify clients when data is available in the space to query.

Reply:
> If you are doing this however, I think writeData needs to be adjusted so it doesn't return an error in the event that no long files exist. 

Good point, I'll do that

Reply:
> My intention was that once the client got a 200 response code, it would mean there was data available for the client to query. The status api will need to change to somehow notify clients when data is available in the space to query.

This seems reasonable, but I'm not not sure 200 conventionally means "some data has been ingested and we'll stream you updates about the rest". I think a HTTP 202 accepted would be the right response, signaling that the job has been accepted for processing. And as written else where, as soon as you get a status update that tells you something is available... then you know something is available.

(As a side note, on the whole this PR tries to follow @mccanne's design referred to in the description; discussion/revisiting of that design should probably involve him).

Reply:
In a future pr we should consider adding back `fsnotify` to kick off the snapshot loop... It would be nice to have the client not wait an arbitrary amount of time on every request to begin querying spaces. That said for now I think this is fine.

Reply:
It's the time of the pcap index, which is very fast. Given that we've gone from "time of full zeek processing" to "time of pcap index", I agree this that further optimization can wait.